com.st.STM32CubeIDE.yaml

```diff
@@ -0,0 +1,72 @@
+app-id: com.st.STM32CubeIDE
+runtime: org.gnome.Platform
```

Comment:
Does it really need GNOME runtime?

Reply:
It runs when using the freedesktop runtime, but it fails to render some menus and gives GTK warnings in the log.

Reply:
The other Eclipse based IDEs (that use SWT) also use the GNOME Platform.